recipes/cartographer/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
+      -G "Ninja" \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=ON \
+      -DBUILD_SHARED_LIBS=ON
+
+cmake --build . --config Release -- -j$CPU_COUNT
```

Comment:
No need for CPU_COUNT with Ninja.
```suggestion
cmake --build . --config Release
```